BLE Indoor Positioning/src/main/java/com/nexenio/bleindoorpositioning/IndoorPositioning.java

```diff
@@ -28,6 +29,11 @@
     public static final long UPDATE_INTERVAL_MEDIUM = 500;
     public static final long UPDATE_INTERVAL_SLOW = 3000;
 
+    public static final double NO_SPEED_FILTER = -1;
+    // set maximum distance to new location
+    private double maximumMovementSpeed = NO_SPEED_FILTER;
```

Comment:
Think from the perspective of someone who is using this library. How can you ever change this value if there's no public setter available?